_data/events/datasaturday0012.yml

```diff
@@ -3,25 +3,26 @@ date: 2021-09-09 08:00:00 -0700
 virtual: true
 description:  "Data Saturday #12 Denver is a free one-day event for professionals who use the Microsoft data platform. 
 This community event offer content across data management, cloud and hybrid architecture, analytics, business intelligence, AI, and more.
-
+<br><br>
 We are planning event to be a hybrid event:
 
-Event location is : Online, platform TBD
 
+Event location is : Online, platform TBD
+<br>
 We look forward to seeing you there.
 "
 registrationurl: https://data-saturday-sofia.eventbrite.com/
 join:
   description: Click on the room you want to join. You can change room at any time to attend the sessions that you prefer.
   rooms:
 
-scheduleurl:
+scheduleurl: https://sessionize.com/api/v2/xzfnfhd6/view/GridSmart
 
-sponsors:
+sponsors: https://datadriventechnologies.org/2021/02/22/2021-data-sql-saturday-sponsorship-plan/
```

Comment:
Sponsore format is

sponsors:
  - link: https://yoursponsorlink
    image: your sponsor image
    height: image height if required